src/main/webapp/features/shared/services/game/Game.service.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+angular.module('catan')
+        .factory('Game', ['GameModel', '$q', 'Remote', function (GameModel, $q, Remote) {
+
+            var Game = function(idOrDetails) {
```

Comment:
but using becomes easier:
- var a = Game(123)    or   var b = Game({....})
instead
- var a = Game(123, null)      or     var b = Game(null, {...})

and what is "status"?